raft_test.go

```diff
@@ -5030,3 +5030,10 @@ func newTestRawNode(id uint64, election, heartbeat int, storage Storage) *RawNod
 	}
 	return rn
 }
+
+// TestPayloadSizeOfEmptyEntry ensures that payloadSize of empty entry is always zero
+func TestPayloadSizeOfEmptyEntry(t *testing.T) {
```

Comment:
Let's move this to `util_test.go`, so that it lives next to the `payloadSize` utility function in `util.go`.